Shell Sort/ShellSortExample.swift

```diff
@@ -8,24 +8,25 @@
 
 import Foundation
 
-public func shellSort(_ list: inout [Int]) {
-    
+public func shellSort(_ list : inout [Int])
+{
     var sublistCount = list.count / 2
-   
-    while sublistCount > 0 {
-        
-        for index in 0..<list.count {
-           
-            guard index + sublistCount < list.count else { break }
+    
+    while sublistCount > 0
+    {
+        for var index in 0..<arr.count{
```

Comment:
I updated the code and checked the format. Hope has fixed the mistake of carelessness.